beacon-chain/sync/initial-sync/service.go

```diff
@@ -127,7 +127,7 @@ func (s *Service) Stop() error {
 
 // Status of initial sync.
 func (s *Service) Status() error {
-	if s.synced.IsNotSet() && s.chainStarted.IsSet() {
```

Comment:
The reason is so that before chainstart, we do not have status errors here. Otherwise while waiting for chainstart you will have a status error on initial sync with this change